cEpiabm/src/simulations/basic_simulation.hpp

```diff
@@ -42,4 +42,4 @@ namespace epiabm
 }
 
 
-#endif // EPIABM_SIMULATIONS_BASIC_SIMULATION_HPP
+#endif // EPIABM_SIMULATINS_BASIC_SIMULATION_HPP
```

Comment:
If this change intentional? I know it is spelt 'SIMULATINS' in a few places now but just wanted to check!